application/listeners/posthelpers.go

```diff
@@ -0,0 +1,129 @@
+package listeners
+
+import (
+	"github.com/silinternational/wecarry-api/domain"
+	"github.com/silinternational/wecarry-api/models"
+	"github.com/silinternational/wecarry-api/notifications"
+)
+
+type msgRecipient struct {
+	nickname string
+	email    string
+}
+
+func getRecipients(eData models.PostStatusEventData) []msgRecipient {
```

Comment:
Is there a way to make this usable by other event listeners?

Reply:
Exported - done.